__tests__/api.test.js

```diff
@@ -151,6 +151,28 @@ describe('POST /api/articles/:article_id/comments', () => {
     })
 })
 
+describe('PATCH /api/articles/:article_id', () => {
+    test('responds with a 200 status code and updated article', () => {
+        const votes = {
+            inc_votes: 1
```

Comment:
Here we need to add three more tests and make sure we are error handling for the following:
Status 400, invalid ID, e.g. string of "not-an-id"
Status 404, non existent ID, e.g. 0 or 9999
Status 400, missing / incorrect body, e.g. inc_votes property is not a number, or missing

Reply:
Hello Hannah, I believe I've done these tests at the bottom of my test suite?